rust/blockstore/src/types/writer_options.rs

```diff
@@ -7,15 +7,21 @@ pub enum BlockfileWriterMutationOrdering {
     Ordered,
 }
 
-#[derive(Debug, Default, Clone, Copy)]
+#[derive(Debug, Clone)]
 pub struct BlockfileWriterOptions {
     pub(crate) mutation_ordering: BlockfileWriterMutationOrdering,
     pub(crate) fork_from: Option<Uuid>,
+    #[allow(dead_code)]
+    pub(crate) prefix_path: String,
 }
 
 impl BlockfileWriterOptions {
-    pub fn new() -> Self {
-        Self::default()
+    pub fn new(prefix_path: String) -> Self {
```

Comment:
this is intentional to force all callers to necessarily think about and set a prefix path. There is no default constructor now.